.github/workflows/dart.yml

```
@@ -173,20 +172,32 @@
+        with:
+          name: badge-data
+          path: badge-data
+
       - name: Prepare docs directory
         run: |
+          # Create directory structure
           rm -rf docs
-          mkdir -p docs
-          cp -R doc/api/* docs/
           mkdir -p docs/coverage docs/badges
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)